### PR TITLE
Mergeordering

### PR DESCRIPTION
**Pull request by DucAnhPhi**
- implemented line ordering algorithm:
 -> order contours by y coordinate
 -> calculate average y deviation between nearest neighbours
 -> separate ordered list into lines where y deviation is greater than the average
 -> order contours in each line by x coordinate
- fix bug where holes in contours are removed

### CODE REVIEW

File: drawing.py [hunk 1]
```diff
@@ -0,0 +1,156 @@
+import cv2 as cv
```
Comment (The-Silverowl) [inline drawing.py]: Do we want to have this file in the finished product?
May be useful for experimenting for the people looking at our code.
We (I) will need to clean it, before then.

File: ordering.py [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+            line = lines[l]
+            line.sort(key=lambda cnt: cnt.center[0])
+            for i in range(len(line)):
+                cnt = line[i]
+                cv.putText(frame, str(l) + str(i), (cnt.center[0], cnt.center[1]),
+                           cv.FONT_HERSHEY_SIMPLEX, 1, (0, 255, 0), 1)
+
+        return lines
+
+    def get_lines_with_hor_vec(self, frame):
```
Comment (The-Silverowl) [inline ordering.py]: We will need to find a good criterium, when to use which mode.

Comment (The-Silverowl) [reply ordering.py]: I will change some small things here.

Comment (DucAnhPhi) [reply ordering.py]: it works best without it at the moment. You are free to improve

**Pull request merged**